packages/react-components/react-jsx-runtime/src/jsx.ts

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import {
+  SlotRenderFunction,
+  UnknownSlotProps,
+  slotInternalChildrenSymbol,
+  slotExternalChildrenSymbol,
+} from '@fluentui/react-utilities';
+
+/**
+ * Equivalent to React.createElement but supporting v9 slot API
+ */
+export function jsx<P extends {}>(
+  type: React.ElementType<P>,
+  props?: P | null,
+  ...children: React.ReactNode[]
+): React.ReactElement<P> | null {
+  return isSlotComponent(type)
+    ? jsxFromSlotComponent(type, props, ...children)
+    : React.createElement(type, props, ...children);
+}
+
+function jsxFromSlotComponent<Props extends UnknownSlotProps>(
+  component: React.ElementType<Props>,
+  props?: Props | null,
+  ...overrideChildren: React.ReactNode[]
+): React.ReactElement<Props> | null {
+  const children = normalizeChildren(props, overrideChildren);
+  const {
+    [slotInternalChildrenSymbol as keyof Props]: _,
+    [slotExternalChildrenSymbol as keyof Props]: externalChildren,
+    ...propsWithoutSymbols
+  } = props ?? {};
```

Comment:
```suggestion
  } = props;
```

Is it possible that `props` are `null` at this stage? If we are there, they should be defined, no?